src/index.html

```diff
@@ -5,9 +5,56 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Stars</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+
```

Comment:
![image](https://github.com/mate-academy/layout_stars/assets/84296041/447bfb77-bb29-420c-9b63-f881a2ddab56)
Remove linebreak between body and first/last child